brozzler/worker.py

```diff
@@ -480,13 +488,22 @@ def _fetch_url(self, site, url=None, page=None):
                 "http": "http://%s" % self._proxy_for(site),
                 "https": "http://%s" % self._proxy_for(site),
             }
+        user_agent = site.get("user_agent")
+        headers = {"User-Agent": user_agent} if user_agent else {}
+        headers.update(site.extra_headers(page))
```

Comment:
I'm surprised we weren't doing this already. We may want to use a library that more closely resembles a browser for `_fetch_url()` in the future, but this is a good step.